### ISSUE TITLE
[feature suggestion] stopwatch step

### ISSUE THREAD

**Issue by NiaJ3oE2LM** [timer-machine/timer-machine-android#111]
This app is very, very good! Well done!

In my current workout there are exercises that require to repeat a specified lap (group of tasks) for as many times as possible within a certain amount of time. For example, a lap may be composed of task A, B, and C and the exercise is to repeat as many laps as possible in 10 minutes. 

With the current version of the app (`7.5.0` from f-droid running on Android 10) I found no way of both running a timer and counting laps. The work around is to use another app (standard stopwatch) to time the execution and count the laps.

If such feature is within the scope of TimeR Machine, you might consider creating a new type of step (in addition to those already available: normal, notification, and group) that behaves like a stopwatch.

**Comment by DeweyReed** [comment]
Thanks for using the app!

To convert a countdown step to a stopwatch, you can add a "Halt" reminder. But your situation requires a stopwatch-like group, which can be a new feature.

**Comment by NiaJ3oE2LM** [comment]
Indeed, the "Halt" function is helpful. Considering the example I was proposing:

+ create a group with enough number of repetitions (set manually so that from a workout point of view, the number of loops corresponds to 10 minutes),
+ inside the group, insert a step for each task A,B, and C.
+ Set each step to 1 second countdown with the "Halt" function so that you need to advance manually

This transforms the group into a simple stopwatch with no memory of past laps and manual setting of the number of laps (which might be unknown). 

It would be nice if TimeR Machine had:

+ the possibility to advance indefinitely on group repetitions without requiring a group loop value. I noticed that the 0 value (group loop) has no special meaning and could be used for such use-case. Of course, this probably requires the addition of a separate button in order to quit the group.
+ an independent time measure relative to the current group, effectively resembling a stopwatch with total running time, number of laps, but no memory of lap duration.

**Comment by Jansherami** [comment]
@DeweyReed is there any chance of just getting a plain stopwatch? It would really be cool and helpful to be able to use stopwatch in the app (besides that I have no complain with this app)

**Comment by DeweyReed** [comment]
@Jansherami Hi, adding a Halt reminder to a step converts it to a stopwatch.

**Comment by Jansherami** [comment]
Oh damns that's soo cool I didn't know that thanks @DeweyReed !
